app/javascript/components/enrollment/steps/AdditionalPlannedTravel.js

```diff
@@ -256,26 +252,11 @@ class AdditionalPlannedTravel extends React.Component {
 }
 
 const schema = yup.object().shape({
-  additional_planned_travel_type: yup
-    .string()
-    .max(200, 'Max length exceeded, please limit to 200 characters.')
-    .nullable(),
-  additional_planned_travel_destination: yup
-    .string()
-    .max(200, 'Max length exceeded, please limit to 200 characters.')
-    .nullable(),
-  additional_planned_travel_destination_country: yup
-    .string()
-    .max(200, 'Max length exceeded, please limit to 200 characters.')
-    .nullable(),
-  additional_planned_travel_destination_state: yup
-    .string()
-    .max(200, 'Max length exceeded, please limit to 200 characters.')
-    .nullable(),
-  additional_planned_travel_port_of_departure: yup
-    .string()
-    .max(200, 'Max length exceeded, please limit to 200 characters.')
-    .nullable(),
+  additional_planned_travel_type: yup.string().max(200, 'Max length exceeded, please limit to 200 characters.').nullable(),
```

Comment:
hmm, even though this results in fewer lines of code, I personally think it's easier to see which validation constraints are in place with the old format

Reply:
I personally like the new way better so it's definitely a personal preference thing

Reply:
I'm with Tim, but there's also no way to have the linter *not* do this. I mentioned it at standup, but this is a bug within `prettier`. If we upgrade to the new prettier version (which we are), this behavior *will* happen. 🤷

Reply:
the pros definitely outweigh the cons IMO